main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -19,6 +19,7 @@ package ca.uwaterloo.flix.language.errors
 import ca.uwaterloo.flix.language.CompilationMessage
 import ca.uwaterloo.flix.language.ast.{Name, SourceLocation, Symbol, UnkindedType}
 import ca.uwaterloo.flix.util.Formatter
+import ca.uwaterloo.flix.util.Grammar.count
```

Comment:
Lets not import it, but use it qualified. That makes it easier to understand where its coming from.